src/include/planner/logical_plan/logical_operator/logical_create_macro.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "base_logical_operator.h"
+#include "function/scalar_macro.h"
+
+namespace kuzu {
+namespace planner {
+
+class LogicalCreateMacro : public LogicalOperator {
+public:
+    LogicalCreateMacro(std::shared_ptr<binder::Expression> outputExpression, std::string macroName,
+        std::unique_ptr<function::ScalarMacro> macro)
+        : LogicalOperator{LogicalOperatorType::CREATE_MACRO}, outputExpression{std::move(
+                                                                  outputExpression)},
+          macroName{std::move(macroName)}, macro{std::move(macro)} {}
+
+    void computeFactorizedSchema() override;
+    void computeFlatSchema() override;
+
+    inline std::shared_ptr<binder::Expression> getOutputExpression() const {
+        return outputExpression;
+    }
+
+    inline std::string getMacroName() const { return macroName; }
+
+    inline std::unique_ptr<function::ScalarMacro> getMacro() const { return macro->copy(); }
+
+    inline std::string getExpressionsForPrinting() const override { return "CREATE " + macroName; }
```

Comment:
then you are printing
`CREATE_MACRO CREATE functionName`

shouldn't we remove "CREATE "+